internal/pkg/skuba/upgrade/addon/versions_test.go

```diff
@@ -124,7 +124,7 @@ func TestHasAddonUpdate(t *testing.T) {
 					kubernetes.Kured:   &kubernetes.AddonVersion{Version: "1.2.0", ManifestVersion: 0},
 					kubernetes.Dex:     &kubernetes.AddonVersion{Version: "2.16.0", ManifestVersion: 0},
 					kubernetes.Gangway: &kubernetes.AddonVersion{Version: "3.1.0", ManifestVersion: 0},
-					kubernetes.PSP:     &kubernetes.AddonVersion{Version: "1.0.0", ManifestVersion: 0},
+					kubernetes.PSP:     &kubernetes.AddonVersion{Version: "1.0.0", ManifestVersion: 1},
```

Comment:
somehow on my local testings, changing this and the one below still makes the tests pass, so this changes may not be necessary or we are not checking this or the tests are wrong. Only the change in line 86 seems to be needed.

Reply:
Ideally these tests shouldn't be tied to the real versions in the `versions.go` file... If they are somehow interdependant we should make them independent.

Reply:
Sorry I made this before :( It would be great if thais PR makes the tests independent to version.go file.

Reply:
I think this patch is pretty important for getting all required pieces for 1.16, we can improve the tests later on a follow-up PR if that's fine.

Reply:
PR for addressing this https://github.com/SUSE/skuba/pull/882 issue.